pennylane/measurements/probs.py

```diff
@@ -299,23 +300,24 @@ def marginal_prob(self, prob, wire_order, batch_size):
         # reshape the probability so that each axis corresponds to a wire
         num_device_wires = len(wire_order)
         shape = [2] * num_device_wires
+        desired_axes = np.argsort(np.argsort(mapped_wires))
```

Comment:
using `np` instead of `qml.math` here because some interfaces don't have `np.insert` which I'm using below when batching

Reply:
Yes I think it should not be an issue because it is only to compute the axes and not the actual array and wires should not be specific to an interface.